src/app/core/models/github/github-comment.model.ts

```diff
@@ -2,10 +2,10 @@ export class GithubComment {
   body: string;
   created_at: string;
   id: number;
-  issue_url: string;
+  issue_url?: string;
   updated_at: string;
-  url: string; // api url
-  user: {
+  url?: string; // api url
```

Comment:
This won't be needed if we use typecasting on our test data

Reply:
Actually for this, I checked the `githubIssue.comments` data that is received under `createPhaseTesterResponseIssue` of `issue.model`, it actually has no user / url attributes. 

![image](https://user-images.githubusercontent.com/59414467/127776789-fd4afb85-71a9-4197-83f7-264410f533e0.png)

Hence I changed it to optional attributes, which I thought would be a more accurate reflection of the actual data we are receiving from GitHub. 

Do you think we should still remove it?

Reply:
I see. Let's take this up in a separate issue, and leave it unchanged here.

Reply:
Sure I will revert these changes